packages/client/screens/MusicPlaylistEditorRoomsSearchScreen/index.tsx

```diff
@@ -80,7 +89,7 @@ const MusicPlaylistEditorRoomsSearchScreen: React.FC<MpeTabMpeRoomsScreenProps>
             >
                 <FlatList
                     testID="library-mpe-room-search-flat-list"
-                    data={libraryRoomState.context.rooms}
+                    data={mpeRooms}
                     renderItem={({ item }) => {
                         return (
```

Comment:
Should we implem the playlist icons for the invited private public etc as for the mtv search cards in this pull request ?

Reply:
Done in the last commit. 👍